config/routes.rb

```diff
@@ -446,6 +446,8 @@ def route_actions_hash
   get("inat_imports/authorization_response",
       to: "inat_imports#authorization_response",
       as: "inat_import_authorization_response")
+  get("inat_imports/cancel/:id", to: "inat_imports#cancel",
+                                 as: "inat_import_cancel")
```

Comment:
By Rails convention, routes that are used to change db values should be `post` or `put` routes. We have a few remaining violations of this convention but i did a lot of work getting rid of them.

Reply:
Making this work is as easy as making this route use `put` instead of `get`, and then using `put_button` instead of `link_to`.